composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "637aa321d05b34fd3af0361c5a1e7b98",
+    "content-hash": "86bf6686ccf1e77bcfb90fa9f0fa5e06",
```

Comment:
Look at https://github.com/gomzyakov/laravel-blog/actions/runs/7162873377/job/19514030450?pr=173


You have an warning on CI:

>Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.